src/Root.cpp

```diff
@@ -423,7 +423,20 @@ void cRoot::LoadWorlds(cSettingsRepositoryInterface & a_Settings, bool a_IsNewIn
 			continue;
 		}
 		FoundAdditionalWorlds = true;
-		cWorld * NewWorld = new cWorld(WorldName.c_str());
+		cWorld * NewWorld;
+		// Note that if there's a world.ini file present, the world will override the following worldTypes in cWorld::start
+		if (WorldName.find("_nether") != std::string::npos)  // If the world has the word nether, mark it as a nether world. (will be overridden by setting  if ini present)
```

Comment:
Ha! Project convention! Very funny comrade.

You could do a C and (nocasecompare == 0), I suppose.


On 3 Feb 2016, at 21:46, "LogicParrot" <notifications@github.com> wrote:

> In src/Root.cpp:
> 
> > @@ -423,7 +423,20 @@ void cRoot::LoadWorlds(cSettingsRepositoryInterface & a_Settings, bool a_IsNewIn
> >  			continue;
> >  		}
> >  		FoundAdditionalWorlds = true;
> > -		cWorld * NewWorld = new cWorld(WorldName.c_str());
> > +		cWorld * NewWorld;
> > +		// Note that if there's a world.ini file present, the world will override the following worldTypes in cWorld::start
> > +		if (WorldName.find("_nether") != std::string::npos)  // If the world has the word nether, mark it as a nether world. (will be overridden by setting  if ini present)
> Do we have a project-specific convention for string comparison? Also, can I easily make this case-insensitive?
> 
> —
> Reply to this email directly or view it on GitHub.
>